mitmproxy/optmanager.py

```diff
@@ -1,21 +1,49 @@
 import contextlib
 import blinker
 import pprint
+import inspect
+import copy
+import functools
+import weakref
+import os
+
+import ruamel.yaml
```

Comment:
There's a C component with a pure-Python fallback. I believe this works correctly, but if you have a Windows box could you check for us?